src/index.js

```diff
@@ -31,13 +32,15 @@ function populateTask(arr) {
     const li = document.createElement('li');
     li.innerHTML = ` 
     <div class  = "d-flex-between">
-      <input type="checkbox" id="${orderArray.index}" value="${orderArray.index}">
+      <input type="checkbox" class="checks" id="${task.index}" value="${task.index}">
       ${task.description}
     </div> 
     <i class="bi bi-three-dots-vertical"></i>`;
     li.classList.add('tasks', 'd-flex-between');
     ul.appendChild(li);
   });
+  getTasks(orderArray);
 }
 
-window.onload(populateTask(tasks));
+populateTask(tasks);
+addListeners();
```

Comment:
- [ ] On page load your tasks don't keep their state in the UI meanwhile the state is correct in the localStorage array. Let's try to address that issue. You can look at your populateTask function and try to add style decoration and color accordingly to the state that you get from the localStorage updated array.